### DISCUSSION TITLE
Ajout d'une heure à toutes les réservations suite à migration 3.4.0b vers 4.0.1

### DISCUSSION THREAD

**Discussion by nic-Oban** [JeromeDevome/GRR#243] [Q&A] [answered]
Bonjour @JeromeDevome @ynaessens ,
nous avons un nouveau souci suite à la migration de version de notre GRR.
Toutes les réservations migrées sont décalées d'une heure en plus. Les nouvelles réservations saisies sont correctement enregistrées.
Je creuse la question plus en profondeur mais est-ce qu'un utilisateur vous aurait remonté ce type de situation ?
Merci par avance et bonne journée

**Comment by JeromeDevome** [comment, marked as answer]
Bonjour @nic-Oban , 
Souvent c'est quand l'heure du serveur est décalé comparé à l'ancien. Par défaut en GTM et en France nous sommes en en GTM +1.

Je présume donc que votre serveur était avant en GTM0

**Reply by nic-Oban** [reply]
Bonjour @JeromeDevome 
ça n'est pas impossible en effet ! Merci pour votre réponse et bonne journée
